src/transaction/log_lsa.hpp

```diff
@@ -56,6 +64,7 @@ struct log_lsa
   inline bool operator<= (const log_lsa &olsa) const;
   inline bool operator> (const log_lsa &olsa) const;
   inline bool operator>= (const log_lsa &olsa) const;
+  inline operator int64_t () const;
```

Comment:
```suggestion
  inline operator std::int64_t () const;
```

Reply:
is there another int64_t besides the standard one? I feel like it only makes the code line longer...

Reply:
consider this just a suggestion and do how you feel is better. in most cases standard functions, classes and types should be used with the std:: namespace. There are exceptions like size_t. int64_t could be one such exception too.